### ISSUE TITLE
About the accuracy of Protonet.

### ISSUE THREAD

**Issue by Glutaredoxin** [Sha-Lab/FEAT#81]
hello, thank you for your excellent work~
I have some problem about accuracy of protonet. when i use Resnet12 as backbone to train protonet from scratch with 30way-1shot, i found the best accuracy is roughly 49%. However, the best acc in your paper can arrive 62%, maybe use pre-train model? in other words, i need pretrain model like traditional CNN in training set and use it in training protonet if i want to get 62% ?
sorry to bother you, I hope I have already clarified the issue thoroughly. :)

**Comment by Han-Jia** [comment]
Hi,

Yes, using the pre-trained weights is an efficient way to obtain high performance with ProtoNet. In detail, a CE loss is optimized on the meta-train set in advance.

If training ProtoNet from scratch, longer epochs and larger ways are required. Please also check the Sufficient Episodic Sampling (SES) trick in [this paper](https://arxiv.org/pdf/2011.14663.pdf) and code from [here](https://github.com/hanlu-nju/revisiting-UML), by resampling sufficient tasks/episodes in a single mini-batch, we can train the Protonet from scratch to around 62%.

**Comment by Glutaredoxin** [comment]
i got it, thank you soso much~ :)

**Comment by ruixuan-gao** [comment]
hi

**Comment by Glutaredoxin** [comment]
Hello, sorry to bother you again~
When i run the pretrain, the val accuracy (16way-1shot) only arrived 26%. could you tell me **the final val acc** in miniimagenet when i use resnet12?

**Comment by Han-Jia** [comment]
Hi, the 16-way 1-shot classification accuracy on the meta-val set should be around 46%. Please check the data augmentation.

For MiniImageNet, the pretrain hyper-parameters are
lr = 0.1, batch-size = 128, max_epoch = 500, schedule: [350, 400, 440, 460, 480], gamma: 0.1
